src/oas3/__tests__/operation.test.ts

```diff
@@ -570,4 +570,99 @@ describe('transformOas3Operation', () => {
       tags: [],
     });
   });
+
+  test('should handle server variables', () => {
```

Comment:
Let's try to make this test a little bit more specific — we can change its title for example:

```suggestion
  test('should keep the server variables', () => {
```

and then instead of comparing the whole operation with an entire gigantic object we could instead make some precise assertions on `operation.server` and checking its variables